doc/nrf/app_dev/device_guides/working_with_nrf/nrf54h/ug_nrf54h20_suit_customize_dfu.rst

```diff
@@ -91,53 +91,65 @@ Variables in the provided templates, like memory ranges and paths to binaries, a
 How to use your own manifest
 ----------------------------
 
-Changing the source of manifest templates allows you to create manifest files on your own, without relying on the templates provided by Nordic Semiconductor.
+When creating an application, it is recommended to create a custom set of manifest templates and use the templates provided by Nordic Semiconductor as a starting point. Using Nordic Semiconductor templates directly in a product is strongly discouraged.
 
-The source of the manifest templates can be configured by setting the following Kconfig options:
+The build system searches for the manifest templates in the following order:
 
-* ``SB_CONFIG_SUIT_ENVELOPE_ROOT_TEMPLATE``
+#. Check if :kconfig:option:`SB_CONFIG_SUIT_ENVELOPE_ROOT_TEMPLATE_FILENAME` or :kconfig:option:`CONFIG_SUIT_ENVELOPE_TEMPLATE_FILENAME` exists at :file:`<sample-dir>/suit/${SB_CONFIG_SOC}/`.
 
-* :kconfig:option:`CONFIG_SUIT_ENVELOPE_TEMPLATE` - For each of the images (application and radio core images)
+#. Check if :kconfig:option:`SB_CONFIG_SUIT_ENVELOPE_ROOT_TEMPLATE_FILENAME` or :kconfig:option:`CONFIG_SUIT_ENVELOPE_TEMPLATE_FILENAME` exists at :file:`<sdk-nrf-dir>/config/suit/templates/${SB_CONFIG_SOC}/${SB_CONFIG_SUIT_BASE_MANIFEST_VARIANT}/`.
 
-One example is demonstrated with the following case:
+The build system selects the set of files from the first successful step.
 
-.. tabs::
+Creating custom manifest templates
+==================================
 
-    .. group-tab:: Windows
+Custom manifest templates can be created in the application directory using the command:
 
-        The provided user-defined manifest templates are stored in ``C:\my_default_templates``.
+.. code-block::
+
+   west suit-manifest init --soc nrf54h20
 
-    .. group-tab:: Linux
+This command will copy the default set of SUIT manifest templates into :file:`suit/nrf54h20` subdirectory. You can edit those manifest templates to your needs and add them into your version control system along with the generated :file:`metadata.yaml` file.
 
-        The provided user-defined manifest templates are stored in ``/home/my_user/my_default_templates``.
+By default, the command uses the manifest templates from the ``default`` variant. Manifest template variants serve different use cases and are described in more detail below.
 
-The following files are used to create the SUIT envelope:
++---------------------------------------------------------------------------------------+-----------------------------------+-----------------------------------+
```

Comment:
Formatting needs to be corrected for line 117 as column headings are not properly formatted. See the preview: https://developer.nordicsemi.com/nRF_Connect_SDK_dev/doc/PR-16222/nrf/app_dev/device_guides/working_with_nrf/nrf54h/ug_nrf54h20_suit_customize_dfu.html#creating-custom-manifest-templates

Reply:
Should be fixed now.